internal/forc/lib/shared.js

```diff
@@ -56,7 +56,10 @@ export const buildFromGitBranch = (branchName) => {
   const stdioOpts = { stdio: 'inherit' };
 
   if (existsSync(swayRepoDir)) {
-    execSync(`cd ${swayRepoDir} && git fetch origin && git checkout ${branchName}`, stdioOpts);
+    execSync(
+      `cd ${swayRepoDir} && git fetch origin && git checkout ${branchName} && git reset --hard origin/${branchName}`,
```

Comment:
Somehow, conflicts emerged between my local sway branch and the origin branch. This addition ensures that the sway repository is always on the origin branch's latest commit.

Reply:
I saw some force pushes happening in the origin branch so that might have caused it.